src/App.js

```diff
@@ -24,8 +24,8 @@ class App extends Component {
 
   render() {
     return (
-      <PersistGate persistor={persistor}>
-        <Provider store={store}>
+      <Provider store={store}>
+        <PersistGate persistor={persistor}>
```

Comment:
was the original ordering incorrect?

Reply:
Yes, as regards to what is contained in the redux persist documentation